### PR TITLE
Custom window size

### PR DESCRIPTION
**Pull request by astrit**
## Description
Resize and center the frontmost window to any custom size.

## Type of change

- [X] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

cws

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/system/custom-window.applescript [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Custom Window Size
+# @raycast.mode silent
+# @raycast.packageName system
```
Comment (unnamedd) [inline commands/system/custom-window.applescript:7]:
```suggestion
# @raycast.packageName System
```

File: commands/system/custom-window.applescript [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# Optional parameters:
+# @raycast.icon ?
```
Comment (unnamedd) [inline commands/system/custom-window.applescript:10]:
```suggestion
# @raycast.icon images/custom-window-size.png
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hey @astritm thanks for you new contribution to our repository!

Just two small comments here! ;)

File: commands/system/custom-window.applescript (hunk 1 above)
Comment (astrit) [reply commands/system/custom-window.applescript]: Updated.

File: commands/system/custom-window.applescript (hunk 2 above)
Comment (astrit) [reply commands/system/custom-window.applescript]: Updated this also, not sure why It happened, initially had a emoji.

**Review by unnamedd — APPROVED** [review]
Now we are good to go! Thanks @astrit!

**Comment by astritm** [comment]
Please tag the right @astrit 😊

**Comment by unnamedd** [comment]
> Please tag the right @astrit 😊

Sorry! My bad! 😳

**Pull request merged**